Mod2/Preparation/Week3/SQLAggregates.md

```diff
@@ -0,0 +1,31 @@
+## Prework for [SQL Aggregates](/Mod2/Lessons/Week3/SQLAggregates.md)
+
+In your notebook, define the following terms:
+1. Minimum
+2. Maximum
+3. Sum
+4. Average
+
+If you are unsure of what these terms refer to, check out the guide below, then redefine them in your own words.
+
+<details><summary>Some Math Terms</summary>
+    <ul>
+        <li><strong>Min:</strong> The first item in an ordered list.  EX: the smallest of a set of numbers.</li>
+        <li><strong>Max:</strong> The last item in an ordered list.  EX: the largest of a set of numbers.</li>
+        <li><strong>Sum:</strong> The total of a collection of numbers. EX: the sum of {1, 3, 5, 4, 2} is 15.</li>
+        <li><strong>Average:</strong> A single number that represents the commonality of a set of numbers. Add up a series of numbers and divide the sum by the total number of values to find the average. EX: the average of {1, 3, 5, 4, 2} is 3.</li>
```

Comment:
Might be helpful to add (1 + 3 + 5 + 4 + 2) / 5 into this description to make average more concrete.